mdal/frmts/mdal_driver.hpp

```diff
@@ -39,13 +40,17 @@ namespace MDAL
 
       virtual bool canRead( const std::string &uri ) = 0;
 
+      //return the maximum vertices per mesh for this driver, default value is big
```

Comment:
Returns (3rd person), missing ! and space. remove "default is big".  I would keep the function as pure virtual and let all drivers implement it (the mesh readers already do it). Or I would keep it as negative number (since some driver even do not have ability to read/write mesh at all)

Reply:
> since some driver even do not have ability to read/write mesh at all

I think it is better to return a negative number for default and for or avoiding to implement this method for all the drivers